client/src/components/App.js

```diff
@@ -21,6 +23,17 @@ import Register from './auth/Register';
 import 'normalize.css';
 import '../styles/styles.scss';
 
+// userData: { username: string, expires: string }
```

Comment:
When the app first loads, check if the user data is stored in localStorage.
If so, check if it's expired or not.

If localStorage contains the user data (username, expires) and it's not expired, set current user